[2016-05-24 04:05:24] requirements_debian_update_system
requirements_debian_update_system () 
{ 
    __rvm_try_sudo apt-get --quiet --yes update || { 
        \typeset __ret=$?;
        case ${__ret} in 
            100)
                rvm_error "There has been error while updating 'apt-get', please give it some time and try again later.
404 errors should be fixed for rvm to proceed. Check your sources configured in:
    /etc/apt/sources.list
    /etc/apt/sources.list.d/*.list
"
            ;;
        esac;
        return ${__ret}
    }
}
current path: /home/chaitanyareddy/.rvm/src/rvm
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/chaitanyareddy/.rvm/bin
command(2): requirements_debian_update_system ruby-2.3.0
++ /scripts/functions/requirements/ubuntu : requirements_debian_update_system()  41 > __rvm_try_sudo apt-get --quiet --yes update
++ /scripts/functions/utility : __rvm_try_sudo()  343 > typeset -a command_to_run
++ /scripts/functions/utility : __rvm_try_sudo()  344 > typeset sudo_path sbin_path missing_paths
++ /scripts/functions/utility : __rvm_try_sudo()  345 > command_to_run=("$@")
++ /scripts/functions/utility : __rvm_try_sudo()  346 > ((  UID == 0  ))
++ /scripts/functions/utility : __rvm_try_sudo()  347 > case "$rvm_autolibs_flag_number" in
++ /scripts/functions/utility : __rvm_try_sudo()  362 > is_a_function __rvm_sudo
++ /scripts/functions/support : is_a_function()  5 > typeset -f __rvm_sudo
++ /scripts/functions/utility : __rvm_try_sudo()  364 > missing_paths=
++ /scripts/functions/utility : __rvm_try_sudo()  365 > for sbin_path in /sbin /usr/sbin /usr/local/sbin
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ -d /sbin ]]
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ :/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/chaitanyareddy/.rvm/bin: != *\:\/\s\b\i\n\:* ]]
++ /scripts/functions/utility : __rvm_try_sudo()  365 > for sbin_path in /sbin /usr/sbin /usr/local/sbin
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ -d /usr/sbin ]]
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ :/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/chaitanyareddy/.rvm/bin: != *\:\/\u\s\r\/\s\b\i\n\:* ]]
++ /scripts/functions/utility : __rvm_try_sudo()  365 > for sbin_path in /sbin /usr/sbin /usr/local/sbin
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ -d /usr/local/sbin ]]
++ /scripts/functions/utility : __rvm_try_sudo()  367 > [[ :/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/chaitanyareddy/.rvm/bin: != *\:\/\u\s\r\/\l\o\c\a\l\/\s\b\i\n\:* ]]
++ /scripts/functions/utility : __rvm_try_sudo()  371 > [[ -n '' ]]
++ /scripts/functions/utility : __rvm_try_sudo()  374 > command_to_run=(__rvm_sudo -p "%p password required for '$*': " "${command_to_run[@]}")
++ /scripts/functions/utility : __rvm_try_sudo()  381 > __rvm_sudo -p '%p password required for '\''apt-get --quiet --yes update'\'': ' apt-get --quiet --yes update
++ /scripts/functions/support : __rvm_sudo()  357 > command sudo -p '%p password required for '\''apt-get --quiet --yes update'\'': ' apt-get --quiet --yes update
++ /scripts/functions/support : __rvm_sudo()  357 > sudo -p '%p password required for '\''apt-get --quiet --yes update'\'': ' apt-get --quiet --yes update
Sorry, try again.
Sorry, try again.
sudo: 3 incorrect password attempts
++ /scripts/functions/utility : __rvm_try_sudo()  381 > return 1
++ /scripts/functions/requirements/ubuntu : requirements_debian_update_system()  43 > typeset __ret=1
++ /scripts/functions/requirements/ubuntu : requirements_debian_update_system()  44 > case ${__ret} in
++ /scripts/functions/requirements/ubuntu : requirements_debian_update_system()  53 > return 1
